flash/image/classification/model.py

```diff
@@ -118,6 +118,7 @@ def __init__(
 
         head = head(num_features, num_classes) if isinstance(head, FunctionType) else head
         head = head or nn.Sequential(
+            nn.Dropout(p=0.1),
```

Comment:
I don't think we should add this here. The classification head just replaces the last layer, so if there was already dropout before it then you would now get: dropout -> dropout -> linear.

Reply:
This is quite un-likely no ? The dropout are usually place within head or deep within the model.

Reply:
Well, most models don't use dropout at all. So you would now be getting a different network architecture. In any case this means that results obtained with the actually architecture cannot be reproduced within Flash.

Reply:
Removed !